reference/opt/shaders-msl/vulkan/vert/multiview.multiview.nocompat.vk.vert

```diff
@@ -0,0 +1,32 @@
+#include <metal_stdlib>
+#include <simd/simd.h>
+
+using namespace metal;
+
+struct MVPs
+{
+    float4x4 MVP[2];
+};
+
+struct main0_out
+{
+    float4 gl_Position [[position]];
+    uint gl_Layer [[render_target_array_index]];
+};
+
+struct main0_in
+{
+    float4 Position [[attribute(0)]];
+};
+
+vertex main0_out main0(main0_in in [[stage_in]], constant uint* spvViewMask [[buffer(24)]], constant MVPs& _19 [[buffer(0)]], uint gl_InstanceIndex [[instance_id]])
+{
+    main0_out out = {};
+    uint spvMaxViewIndex = 31 - clz(*spvViewMask);
+    uint gl_ViewIndex = gl_InstanceIndex % spvMaxViewIndex;
+    gl_InstanceIndex /= spvMaxViewIndex;
```

Comment:
How do we handle indirect here? I assume the indirect buffer must be modified by multiplying by number of views?

Reply:
Is it possible to unroll the multiview by just emitting N draw calls? I imagine the implementation for that would be far simpler than fixing this issue.